install_config/topics/ocp_osp_provisioning.adoc

```diff
@@ -0,0 +1,271 @@
+[[inventory_provision]]
+=== Preparing the Inventory for Provisioning
+
+With the installation of the `openshift-ansible` package complete via our
+previous steps, there resides a
+`sample-inventory` directory that we will copy to our `cloud-user` home directory
+of the deployment host.
+
+On the deployment host,
+
+----
+$ cp -r /usr/share/ansible/openshift-ansible/playbooks/openstack/sample-inventory/ ~/inventory
+----
+
+Within this inventory directory, the _all.yml_ file contains all the different
+parameters that must be set in to order to achieve successful provisioning of
+the RHOCP instances. The _OSEv3.yml_ file contains some references required by
+the _all.yml_ file plus all the different OpenShift cluster parameter installation
+customization one would want.
+
+[[all.yml]]
+==== all.yml
```

Comment:
@ahardin-rh - Will need to change this to something other than all.yml (for both the anchor and the title) as the customer portal doesn't build if left like this.